src/components/ProfileContent/ProfileBalance/ProfileBalance.tsx

```diff
@@ -39,14 +41,18 @@ const ProfileBalance = ({ advertiserStats }: { advertiserStats: DeepPartial<TAdv
         ]
     );
 
-    const labelSize = isMobile ? 'md' : 'sm';
+    const labelSize = isMobile ? 'xl' : 'md';
 
     return (
         <>
             <AvailableP2PBalanceModal
                 isModalOpen={shouldShowAvailableBalanceModal}
                 onRequestClose={() => setShouldShowAvailableBalanceModal(false)}
             />
+            <RemainingBuySellLimitModal
+                isModalOpen={shouldShowRemainingBuySellLimitModal}
+                onRequestClose={() => setShouldShowRemainingBuySellLimitModal(false)}
+            />
```

Comment:
how come we don't use useModalManager's showModal, hideModal and isModalOpenFor here for these two modals?

Reply:
oh yeah this needs to be refactored